src/applications/apps.js

```diff
@@ -251,7 +251,7 @@ export function validateRegisterWithConfig(config) {
   if (Array.isArray(config) || config === null)
     throw Error(
       formatErrorMessage(
-        31,
```

Comment:
Ah, we missed the error code numbers - https://single-spa.js.org/error/?code=31 is for a different one.

Thanks for the fix.